protocol/v2/ssv/validator/committee.go

```diff
@@ -87,7 +87,7 @@ func (c *Committee) RemoveShare(validatorIndex phase0.ValidatorIndex) {
 func (c *Committee) StartConsumeQueue(logger *zap.Logger, duty *spectypes.CommitteeDuty) error {
 	c.mtx.Lock()
 	defer c.mtx.Unlock()
-
+	logger.Info("Starting committee consume queue")
```

Comment:
this is unnecessary as there's already a starting log inside ConsumeQueue

Reply:
Removed 👌